runelite-api/src/main/java/net/runelite/api/VarPlayer.java

```diff
@@ -227,6 +227,13 @@
 	 */
 	LAST_HOME_TELEPORT(892),
 
+	/**
+	 * Charge spell
+	 * 0 < Value * 2 = Remaining ticks on charge buff for God spells
```

Comment:
Wut? Is this meant to convey that values above 0 are double the number of remaining ticks? ie. `10` represents 5 remaining ticks? This is not clear at all.